test/amb_native_to_erc20/foreign_mediator.test.js

```diff
@@ -1561,17 +1562,15 @@ contract('ForeignAMBNativeToErc20', async accounts => {
       const erc20Token = await ERC20Mock.at(token.address)
       const transferEvents = await getEvents(erc20Token, { event: 'Transfer' })
       // 5 transfer events: 1 Mint to user, 1 Mint to fee manager, fee manager 1 transfer to each reward account,
-      // 1 feeReceiver contract transfer to mediator, 1 transfer to 0 to burn the tokens
-      expect(transferEvents.length).to.be.equal(7)
+      // bridge should not allow to send messages in backward direction, so fee manager should not be able to send tokens through the bridge
```

Comment:
Could you clarify how this test will be changed after https://github.com/poanetwork/tokenbridge-contracts/issues/424 addressing?

Reply:
I think it should be left as it is now. This restriction for multiple calls allowed at most 1 message per transaction. As for now, passing new messages while processing other message is not allowed at all. So, removing that check in #424 should not change the contract behaviour.

Reply:
ok. thanks